composer.json

```diff
@@ -36,7 +36,7 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "respect/data": "0.2.2"
+        "respect/data": "~0.2"
```

Comment:
I think it can break Relational easily. `0.2.*` or `~0.2.2` are better in this case.